packages/documentation/README.md

```diff
@@ -8,21 +8,21 @@ This website is built with [Starlight](https://starlight.astro.build/), a docume
 
 ### Local Development
 
-- Make sure all the dependencies for the website are installed:
+- Make sure all the dependencies for the website are installed. Because this is a monorepo, you should run the following command in the root of the project folder:
```

Comment:
Maybe something like "should run the following in _this_ project folder (/packages/documentation)" to differentiate from the "root"

Reply:
I just assumed the person would install the dependencies throughout the repo. Because when we make changes to the dependencies on the repo, the pnpm-lock is updated outside the repo anyway.